### PR TITLE
Feature/a2 1290 lpa questionnaire

### PR DESCRIPTION
**Pull request by Lpsatti**
## Ticket Number
https://pins-ds.atlassian.net/browse/A2-1290

## Description of change

LPA Questionnaire Automation tests

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ X] My commit history in this PR is linear
- [X ] New features have tests
- [ ] Breaking change (team conversation required)

### CODE REVIEW

File: test-packages/platform-feature-tests/cypress/support/commands.js [hunk 1]
```diff
@@ -99,3 +109,16 @@ Cypress.Commands.add('uploadFileFromFixtureDirectory', (filename) => {
 	})
 	cy.get('input[type="file"]').selectFile(`cypress/fixtures/${filename}`, { force: true });
 });
+
+
+Cypress.Commands.add('checkIfUnchecked',(labelText) => {
+	cy.contains('label',labelText).invoke('attr','for').then((id) => {
+		cy.get(`#${id}`).then($checkbox =>{
+			if(!$checkbox.is(':checked')){
+				cy.wrap($checkbox).click();}
```
Comment (PeterBridgerKC) [inline test-packages/platform-feature-tests/cypress/support/commands.js:118]: A minor point, but the formatting is a little confusing here in terms of line breaks and where the curly braces sit

File: test-packages/platform-feature-tests/cypress/support/flows/sections/lpaManageAppeals/houseHolderQuestionnaire.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	const siteAccess = new SiteAccess();
+	const notifyParties = new NotifyParties();
+	const poReportAndSupportDocs = new PoReportAndSupportDocs();
+	let appealId;
+	let counter = 0;
+	cy.get(basePage?._selectors.trgovukTableRow).each(($row) => {
+		const rowtext = $row.text();
+		if (rowtext.includes(lpaManageAppealsData?.hasAppealType) && !rowtext.includes(lpaManageAppealsData?.todoInvalid)) {
+			if (counter === 0) {
+				cy.log(rowtext);
```
Comment (PeterBridgerKC) [inline test-packages/platform-feature-tests/cypress/support/flows/sections/lpaManageAppeals/houseHolderQuestionnaire.js:27]: Just checking you want to leave in this `cy.log`

File: test-packages/platform-feature-tests/cypress/support/flows/sections/lpaManageAppeals/houseHolderQuestionnaire.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+				cy.wrap($row).within(() => {
+					cy.get(basePage?._selectors.trgovukTableCell).contains(lpaManageAppealsData?.hasAppealType).should('be.visible');
+					cy.get('a').each(($link) => {
+						if ($link.attr('href')?.includes(lpaManageAppealsData?.todoQuestionnaire)) {
+							appealId = $link.attr('href')?.split('/').pop();
+							cy.log(appealId);
```
Comment (PeterBridgerKC) [inline test-packages/platform-feature-tests/cypress/support/flows/sections/lpaManageAppeals/houseHolderQuestionnaire.js:34]: Again with this `cy.log`

File: test-packages/platform-feature-tests/cypress/page-objects/appeal-information-upload/appeal-site-address.js [hunk 4]
```diff
@@ -148,7 +150,7 @@ export class AppealSiteAddress {
 		}
 	}
 
-	healthAndSafteyIssues() {
+	healthAndSafteyIssues(option) {
```
[the remark this answers is not in the document]
Comment (Lpsatti) [reply test-packages/platform-feature-tests/cypress/page-objects/appeal-information-upload/appeal-site-address.js:153]: yes, it is old code. Removed to resolve error

File: test-packages/platform-feature-tests/cypress/support/commands.js (hunk 1 above)
Comment (Lpsatti) [reply test-packages/platform-feature-tests/cypress/support/commands.js]: Indentation applied

File: test-packages/platform-feature-tests/cypress/support/flows/sections/lpaManageAppeals/houseHolderQuestionnaire.js (hunk 2 above)
Comment (Lpsatti) [reply test-packages/platform-feature-tests/cypress/support/flows/sections/lpaManageAppeals/houseHolderQuestionnaire.js]: removed log

File: test-packages/platform-feature-tests/cypress/support/flows/sections/lpaManageAppeals/houseHolderQuestionnaire.js (hunk 3 above)
Comment (Lpsatti) [reply test-packages/platform-feature-tests/cypress/support/flows/sections/lpaManageAppeals/houseHolderQuestionnaire.js]: removed log

**Review by bryn-pins — APPROVED** [review]

**Pull request merged**